lambda/lambda.js

```diff
@@ -0,0 +1,88 @@
+async function authenticate() {
+  const baseUrl = process.env.SANCTUARY_API_BASE_URL || "localhost";
+  const path = process.env.SANCTUARY_API_AUTH_PATH || "/api/auth/";
+  const email = process.env.SANCTUARY_EMAIL || "one@test.com";
+  const password = process.env.SANCTUARY_PASSWORD || "sanctuary";
+
+  const data = JSON.stringify({
+    email,
+    password
+  })
+  const options = {
+    hostname: baseUrl,
+    port: 80,
+    path,
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      "Content-Length": data.length
+    }
+  };
+  
+  return new Promise((resolve, reject) => {
+    const lib = baseUrl.startsWith("https") ? require("https") : require("http");
+    const req = lib.request(options, (res) => {
+      let rawData = "";
+
+      res.on("data", (chunk) => { rawData += chunk; });
+
+      res.on("end", () => resolve(JSON.parse(rawData)));
+    })
+    
+    req.on("error", (error) => reject(error))
+
+    req.write(data);
+    req.end();
+  });
+}
+
+async function triggerSendMessages (authToken) {
+  const baseUrl = process.env.SANCTUARY_API_BASE_URL || "localhost";
+  const path = process.env.SANCTUARY_API_SEND_MESSAGE_PATH || "/api/user/me";
+  
+  const options = {
+    hostname: baseUrl,
+    path,
+    headers: {
+        Authorization: `Bearer ${authToken}`
+    }
+}
+
+  return new Promise((resolve, reject) => {
+    const lib = baseUrl.startsWith("https") ? require("https") : require("http");
+    const request = lib.get(options, (res) => {
+      if (res.statusCode < 200 || res.statusCode > 299) {
+        return reject(`SendMessage API Call Failed with Code: ${res.statusCode}`);
+      }
+      // Not interested in the response.
```

Comment:
could we turf the response off to cloudwatch or something? we don't currently have any visibility into failed messages.

Reply:
could handle in the API just as easily though if we started a log account somewhere or something.

Reply:
We could but that means the lambda might sit waiting and running for the API to process all those messages and send a response

Reply:
Oh true. Alright, we'll do it somewhere else.